src/librustc/middle/ty.rs

```diff
@@ -4412,10 +4412,6 @@ impl<'tcx> TyS<'tcx> {
         result
     }
 
-    pub fn is_ffi_safe(&'tcx self, cx: &ctxt<'tcx>) -> bool {
-        !self.type_contents(cx).intersects(TC::ReachesFfiUnsafe)
-    }
-
```

Comment:
Does this mean we no longer use `ReachesFfiUnsafe` at all? If so, it should be removed, not just this helper function.